webpack/assets/javascripts/react_app/components/notifications/notifications.scss

```diff
@@ -36,6 +36,27 @@
       position: static;
     }
 
+    .drawer-pf-close,
+    .drawer-pf-toggle-expand {
+      color: inherit;
+      cursor: pointer;
+      line-height: inherit;
+      padding: 2px 10px;
+      position: absolute;
+
+      &:hover {
+        text-decoration: none;
+      }
+    }
+
+    .drawer-pf-close {
+      right: 0;
+    }
+
+    .drawer-pf-toggle-expand {
+      left: 0;
+    }
+
```

Comment:
@ohadlevy @sharvit is this the styling that had to be pulled in manually?

Reply:
@waldenraines - yes
we just copied them from here:
https://github.com/patternfly/patternfly/blob/master/src/less/notifications-drawer.less#L74-L98

I also saw that the `&:focus` is missing so i just added it with another commit.

Reply:
As @ohadlevy say, the whole file should be removed after the upgrade.

It doesn't look like `patternfly` is working on a `bootstrap-v4` version, they do have a `v4.0.0-rc.1` tag but it is outdated and it is based on `boostrap-v3`.
https://github.com/patternfly/patternfly/tree/v4.0.0-rc.1

I think that a better way will be to move all the patternfly related files from webpack assets to the app assets.
Then we should split the scss to `patternfly-versions-gap` and `patternfly-extensions`

We should update `patternfly-versions-gap` every time we facing issues of outdated code.
Then we should remove the relevant code when updating `patternfly-sass`, the file itself should contain only `less to scss` converted code with comments that contain the version number and lead to the original `patternfly` code.

Reply:
@sharvit breaking down assets sound like a good plan, can you do it please in a separate PR? 

Is there any reason to move it from webpack back to the asset pipeline? I don't see much value in that since our long term goal is to minimize the usage of the asset pipeline.
the only reason I can think of if it hurts any existing themes / branding people might be using /cc @ShimShtein

Reply:
@ohadlevy - Yes, I am going to focus on that right now.

The only reason I don't want to have a patternfly code in the webpack assets is because patternfly itself is not a part of the webpack assets. it will make the "world of patternfly" more readable because all the imports related to patternfly will be in one place.